### PR TITLE
Created a counter function for users to request verification

### PR DESCRIPTION
**Pull request by Zer0SZN**

### CODE REVIEW

**Comment by Zer0SZN** [comment]
Someone should review my PR

File: VerificationCounterFunction/RequestVerificationLink.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const requestVerificationLink = () => {
+  // Disable the button to prevent multiple requests
+  document.getElementById("requestLinkButton").disabled = true;
```
Comment (annonymousauthority) [inline VerificationCounterFunction/RequestVerificationLink.js:3]: Please remove this

File: VerificationCounterFunction/RequestVerificationLink.js
Comment (annonymousauthority) [inline VerificationCounterFunction/RequestVerificationLink.js:1]: The entire file doesn't do the work.

Please connect with @Ibhassan01 and @7malikk for help

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: VerificationCounterFunction/RequestVerificationLink.js [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
+  const intervalID = setInterval(() => {
+    counter--;
+    if (counter == 0) {
+      clearInterval(intervalID);
+      return true;
     }
   }, 1000);
+  return false;
 };
+startCounter();
```
Comment (annonymousauthority) [inline VerificationCounterFunction/RequestVerificationLink.js:12]: We don't need this

Rather do a named export 

`export {startCounter};`

File: VerificationCounterFunction/RequestVerificationLink.js
Comment (annonymousauthority) [reply VerificationCounterFunction/RequestVerificationLink.js:1]: Please consult @Remi-dee for a way out of this function

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: VerificationCounterFunction/RequestVerificationLink.js (hunk 2 above)
Comment (Remi-dee) [reply VerificationCounterFunction/RequestVerificationLink.js:12]: Hi @Zer0SZN, you can turn your function to a named export like so:

```
function startCounter() {
  let counter = 120;
  const intervalID = setInterval(() => {
    counter--;
    if (counter == 0) {
      clearInterval(intervalID);
      return true;
    }
  }, 1000);
  return false;
};

export {startCounter};
```

Comment (Zer0SZN) [reply VerificationCounterFunction/RequestVerificationLink.js:12]: Yea thanks bro

**Review by Remi-dee — COMMENTED** [review]
All good.

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: VerificationCounterFunction/RequestVerificationLink.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  let counter = 120;
+  const timer = setInterval(() => {
+    if (counter == 0) {
+      clearInterval(timer);
+      return true;
+    } else {
+      counter--;
+    }
+  }, 1000);
+  return false;
```
[the remark this answers is not in the document]
Comment (Zer0SZN) [reply VerificationCounterFunction/RequestVerificationLink.js:11]: Ok

[the remark this answers is not in the document]
Comment (Zer0SZN) [reply VerificationCounterFunction/RequestVerificationLink.js]: Done

File: VerificationCounterFunction/RequestVerificationLink.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+function startCounter() {
+  let counter = 120;
+  const timer = setInterval(() => {
+    if (counter == 0) {
+      clearInterval(timer);
+      return true;
```
Comment (annonymousauthority) [inline VerificationCounterFunction/RequestVerificationLink.js:6]: Change this to `callback(true)`

File: VerificationCounterFunction/RequestVerificationLink.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    } else {
+      counter--;
+      return false;
```
Comment (annonymousauthority) [inline VerificationCounterFunction/RequestVerificationLink.js:9]: change this to `callback(false)`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Review by annonymousauthority — APPROVED** [review]
Updates can be made on the Trello board

**Pull request merged**